packages/uswds-core/src/styles/mixins/helpers/at-media.scss

```diff
@@ -5,9 +5,11 @@
 
 // Mobile-first media query helper
 
+$our-breakpoints: map-deep-get($system-properties, breakpoints, standard);
+$custom-breakpoints: map-deep-get($system-properties, breakpoints, extended);
+
```

Comment:
**note:** Moved variable declaration out of mixin since they are used in both `at-media` and `at-media-max`